torchaudio/_internal/module_utils.py

```diff
@@ -77,6 +77,34 @@ def wrapped(*args, **kwargs):
     return decorator
 
 
+def is_soundfile_available():
```

Comment:
Since this function will be called multiple times, I think it's better to avoid repeating `try:import~except:~` in it.

I think something like the following should do.

```python
def _check_soundfile_importable():
    try:
        import soundfile
        return True
    except Exception:
        return False


_is_soundfile_importable = _check_soundfile_importable()


def is_soundfile_available():
    return is_module_available('soundfile') and _is_soundfile_importable
```

Reply:
Great point, updated, I still need to check module_available in _check_soundfile_importable b/c we wanted to avoid import if module not exist.